rest_sensors.cpp

```diff
@@ -1167,8 +1167,8 @@ int DeRestPluginPrivate::changeSensorConfig(const ApiRequest &req, ApiResponse &
                 }
                 else if (rid.suffix == RConfigCoolSetpoint)
                 {
-                    if (map[pi.key()].type() == QVariant::Double)
-                    {
+                    //if (map[pi.key()].type() == QVariant::Double)
+                    //{
                         qint16 coolsetpoint = map[pi.key()].toInt(&ok);
```

Comment:
I wouldn't remove that check. If the client sends the value as string it should be fixed in the client by using a number.

The docs also describe that parameter as Number (700–3500):

https://dresden-elektronik.github.io/deconz-rest-doc/endpoints/sensors/